src/scripts/Sitemap.js

```diff
@@ -4,7 +4,15 @@ import SelectorList from './SelectorList';
 import Model from './Model';
 
 export default class Sitemap {
-	constructor(id, startUrls, model, selectors) {
+	constructor(id, startUrls, model, selectors, extensionVersion = null) {
+		this.rootSelector = { id: '_root', uuid: '0' };
+		this.extensionVersion =
```

Comment:
`extensionVersion` is a bit confusing because it resembles a version of plugin, e.g. 0.3.717, which is ok to use here, but if you'd like to use separate counter (0, 1, ...), I'd suggest calling it [sitemap] specification version or something like that.